doc/tutorials/navigation2_on_real_turtlebot3.md

```diff
@@ -0,0 +1,116 @@
+# Navigation2 on a Real Turtlebot 3
+
+- [Overview](#overview)
+- [Requirments](#requirements)
+- [Navigation2 with a real Turtlebot 3](#navigation2-with-a-real-turtlebot-3)
+- [Videos](#videos)
+
+# Overview:
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot. Before completing this tutorials, completing the Navigation2 with Turtlebot 3 in Gazebo is highly recommended espesially if you are new to ROS and Navigation2.
+
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2). Turtblebot robots are widely supported by the ROS community. You can find many applications and examples of Turtlebot projects on the Internet. You can find more information about Turtlebot3 [here.](http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/)
+
+Links to the robot kits and Github repo.
+
+- [Turtlebot Burger](http://www.robotis.us/turtlebot-3-burger-us/)
+- [Turtlebot Waffle](http://www.robotis.us/turtlebot-3-waffle-pi/)
+- [Turtlebot 3 Github](https://github.com/ROBOTIS-GIT/turtlebot3)
+
+This tutorial consists of two parts. In the first part, you will learn how to use Turtlebot 3 robots in simulation (Gazebo). And in the second part, you will learn how to control a real Turtlebot Waffle using Navigation2.
```

Comment:
```suggestion
This tutorial consists of two parts. In the first part, you learned how to use Turtlebot 3 robots in simulation (Gazebo). And in this part, you will learn how to control a real Turtlebot Waffle using Navigation2.
```

Also, I'd add a link to the other file everywhere you refer to it. eg. `in the [first part](navigation2_with_turtlebot3_in_gazebo.md)